dev/tests/integration/framework/Magento/TestFramework/Helper/Amqp.php

```diff
@@ -13,26 +13,57 @@
  */
 class Amqp
 {
+    const CONFIG_PATH_QUEUE_AMQP_HOST = 'queue/amqp/host';
+    const CONFIG_PATH_QUEUE_AMQP_USER = 'queue/amqp/user';
+    const CONFIG_PATH_QUEUE_AMQP_PASSWORD = 'queue/amqp/password';
+
     /**
      * @var Curl
      */
     private $curl;
 
+    /**
+     * @var \Magento\Framework\App\DeploymentConfig
+     */
+    private $deploymentConfig;
+
     /**
      * RabbitMQ API host
      *
      * @var string
      */
-    private $host = 'http://localhost:15672/api/';
+    private $host;
 
     /**
      * Initialize dependencies.
+     * @param \Magento\Framework\App\DeploymentConfig $deploymentConfig
      */
-    public function __construct()
-    {
+    public function __construct(
+        \Magento\Framework\App\DeploymentConfig $deploymentConfig
```

Comment:
Please inject new dependencies in backward-compatible manner: https://devdocs.magento.com/guides/v2.3/contributor-guide/backward-compatible-development/#adding-a-constructor-parameter